package.json

```diff
@@ -19,19 +19,22 @@
     "react-component"
   ],
   "peerDependencies": {
-    "react": "~0.13.3"
+    "react": "~0.14.6"
   },
   "dependencies": {
     "classnames": "^1.2.0",
     "es5-shim": "^4.0.3",
+    "fbjs": "^0.6.1",
```

Comment:
FYI you might want to pin to the exact version here, fbJS warns that they [may not follow semver](https://github.com/facebook/fbjs#purpose)